src/main/java/controller/planning/EventPlanningController.java

```diff
@@ -47,25 +51,31 @@ private void saveEventInDatabase() {
 
         createPartyInDatabase(eventName, eventDate);
         createUsersInDatabase(users);
+        createPartitipationsInDatabase(users_id, party_id);
     }
 
     private void createPartyInDatabase(String eventName, LocalDateTime eventDate){
         DBConnectionProvider postgresqlConnectionProvider = new PostgresqlConnectionProvider();
+        party_id = null;
+        Party tempParty = new Party();
         try{
             partyRepository = new PartyRepository(postgresqlConnectionProvider.getConnection());
```

Comment:
Why is field if it's recreated each time `createPartyInDatabase` is called? I suggest the field to stay, but create it only once.